src/engine/Server.cpp

```diff
@@ -770,19 +781,13 @@ CPP_template_2(typename RequestT, typename ResponseT)(
     requires ad_utility::httpUtils::HttpRequest<RequestT>)
     Awaitable<void> Server::processQuery(
         const ad_utility::url_parser::ParamValueMap& params,
-        ParsedQuery&& query, const ad_utility::Timer& requestTimer,
+        PlannedQuery&& plannedQuery, const ad_utility::Timer& requestTimer,
```

Comment:
I think you can now just use a const-reference here and avoid the move operation at the caller